code/modules/projectiles/projectile/bullets/shotgun.dm

```diff
@@ -79,7 +78,7 @@
 
 /obj/item/projectile/bullet/pellet/shotgun_improvised
 	tile_dropoff = 0.55		//Come on it does 6 damage don't be like that.
-	damage = 6
+	damage = 12
```

Comment:
It's not called improvised shell for nothing, but I do love me the effects on it